src/clib/lib/job_queue/slurm_driver.cpp

```diff
@@ -409,7 +409,7 @@ void *slurm_driver_submit_job(void *__driver, const char *cmd, int num_cpu,
     auto submit_script =
         make_submit_script(driver, cmd, job_name, num_cpu, argc, argv);
     std::vector<std::string> sbatch_argv = {
-        "--workdir=" + std::string(run_path),
+        "-D=" + std::string(run_path),
```

Comment:
This does not work on the TNO SLUM clusters.

I tested `--chdir` on two SLURM systems that we use at TNO, and that works fine.

Hence, I suggest changing it to `--chdir`.